twitter-sentiment/src/app.py

```diff
@@ -1,33 +1,24 @@
+import os
 from typing import Tuple
 from h2o_wave import Q, app, ui, main, data
 
 from .tweet_analyser import TweetAnalyser
 
+env_vars=[]
```

Comment:
This is a global app variable, but we add to it in the `init` function which is called for every new browser that goes to the app. So on my local machine if I have two tabs open at once for the app the value of `env_vars` is `[None, None, None, None, None, None, None, None]` rather than `[None, None, None, None]`. 

Since all users in the cloud share the environmental variables it would be better to set them at the `q.app` level instead I think.